src/sentry/utils/email.py

```diff
@@ -10,22 +10,24 @@
 import logging
 import os
 import time
-import toronado
+from email.utils import parseaddr
+from operator import attrgetter
 from random import randrange
 
+import toronado
```

Comment:
Yeah, this is how it's supposed to work from PEP 8, this a "related third party import": https://www.python.org/dev/peps/pep-0008/#imports